aspnetcore/blazor/globalization-localization.md

```diff
@@ -363,8 +363,12 @@ CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
 CultureInfo.DefaultThreadCurrentUICulture = new CultureInfo("en-US");
 ```
 
+:::moniker range="< aspnetcore-10.0"
+
 > [!NOTE]
-> Currently, Blazor WebAssembly apps only load resources based on <xref:System.Globalization.CultureInfo.DefaultThreadCurrentCulture>. For more information, see [Blazor WASM only relies on the current culture (current UI culture isn't respected) (`dotnet/aspnetcore` #56824)](https://github.com/dotnet/aspnetcore/issues/56824).
+> In .NET 9 or earlier, standalone Blazor WebAssembly apps load UI internationalization resources based on <xref:System.Globalization.CultureInfo.DefaultThreadCurrentCulture?displayProperty=nameWithType>. To set the default UI culture using <xref:System.Globalization.CultureInfo.DefaultThreadCurrentUICulture?displayProperty=nameWithType> in a standalone Blazor WebAssembly app, [upgrade the app to .NET 10 or later](xref:migration/index).
```

Comment:
These sentences are correct but might be interpreted as:
- in <= net9, use `CurrentCulture` for globalization
- in > net9, use `CurrentUICulture` for globalization.
Which is not what we want to convey.

Could you convert this proposal into doc-friendly form:

`
In .NET 9 or earlier, standalone Blazor WebAssembly apps load UI globalization resources based on <xref:System.Globalization.CultureInfo.DefaultThreadCurrentCulture?displayProperty=nameWithType>. If you want to additionally load globalization data for your localization culture defined by <xref:System.Globalization.CultureInfo.DefaultThreadCurrentUICulture?displayProperty=nameWithType>, [upgrade the app to .NET 10 or later](xref:migration/index).
`